cmake/projects/nsync/hunter.cmake

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2016-2017, Ruslan Baratov
+# Copyright (c) 2017, David Hirvonen
+# All rights reserved.
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_cmake_args)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    nsync
+    VERSION
+    1.14-p0
+    URL
+    "https://github.com/hunter-packages/nsync/archive/v1.14-p0.tar.gz"
```

Comment:
Use this release: https://github.com/hunter-packages/nsync/releases/tag/v1.14-p1

Reply:
Ouch.  Thanks.  Done. Test == `SUCCESS`